doc/dev/mfx_trace2-dev.md

```diff
@@ -0,0 +1,73 @@
+# MediaSDK tracing (mfx_trace2). Developer documentation
+
+## Overview
+
+See [mfx_trace2 user documentation](./mfx_trace2.md).
+
+Main trace system object is presented by global instance of `mfx::Trace` class (instance is called `_mfx_trace`).
+Its constructor (`mfx::Trace::Trace()`) reads configuration file (on Linux) or registry values (on Windows), initializes all enabled backends.
+
+User is not supposed to touch internals of `mfx::Trace` class except `mfx::Trace::Scope` which is public interface for user to enable scoped tracing. `mfx::Trace::pushEvent(const Trace::Event &e)` function stores event in std::vector `_mfx_trace.events` and calls `handleEvent(const Trace::Event &e)` function for every enabled backend.
+
+Event saves the following information:
+- SourceLocation (`mfx::SourceLocation` class)
```

Comment:
Add a note somewhere that this mfx::SourceLocation will be replaced w/ https://en.cppreference.com/w/cpp/utility/source_location

Reply:
Done